Tests/ASTableViewThrashTests.mm

```diff
@@ -47,10 +47,10 @@ - (void)tearDown
 }
 
 // NOTE: Despite the documentation, this is not always called if an exception is caught.
-- (void)recordFailureWithDescription:(NSString *)description inFile:(NSString *)filePath atLine:(NSUInteger)lineNumber expected:(BOOL)expected
+- (void)recordIssue:(XCTIssue *)issue
 {
   _failed = YES;
-  [super recordFailureWithDescription:description inFile:filePath atLine:lineNumber expected:expected];
```

Comment:
oops, it looks like this change is Xcode 12+ only, so I may need to tweak it for this PR, because CI is running on Xcode 11.5 currently.